controller/wingsail/wingsail_ctrl_node.py

```diff
@@ -22,7 +23,8 @@ class WingsailControllerNode(Node):
     while optimizing for speed by maximizing the lift-to-drag ratio of the wingsail.
 
     Subscriptions:
-        TO BE ADDED
+        __filtered_wind_sensors_sub (Subscription): Subscribes to a `WindSensor` message.
+        __gps_sub (Subscription): Subscribes to a 'GPS' message.
```

Comment:
Rather than putting the topic type, put the topic name instead. This is more specific because the topic type could refer to a variety of different topics.

Reply:
changed it to:
> __filtered_wind_sensors_sub (Subscription): Subscribes to the filtered_wind_sensor topic
>  __gps_sub (Subscription): Subscribes to the gps topic

Resolve this if you agree with the change.